Adyen/Model/BalanceControl/AbstractOpenAPISchema.cs

```diff
@@ -0,0 +1,78 @@
+/*
+* Adyen Balance Control API
+*
+*
+* The version of the OpenAPI document: 1
+* Contact: developer-experience@adyen.com
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+
+using System;
+using Newtonsoft.Json;
```

Comment:
weren't we going to move away from Newtonsoft?

Reply:
not anytime soon :(